app/views/stages/_fields.html.erb

```diff
@@ -28,7 +28,7 @@
       <%= form.label :name, 'Deploy Groups', class: 'col-lg-2 control-label' %>
       <div class="col-lg-4">
         <%= form.collection_select(
```

Comment:
no need to add includes anymore ;)

On Tue, Mar 17, 2015 at 7:17 PM, Jon Moter <notifications@github.com> wrote:

> In app/views/stages/_fields.html.erb
> <https://github.com/zendesk/samson/pull/302#discussion_r26600368>:
>
> > @@ -28,7 +28,7 @@
> >        <%= form.label :name, 'Deploy Groups', class: 'col-lg-2 control-label' %>
> >        <div class="col-lg-4">
> >          <%= form.collection_select(
>
> As a rule of thumb, I dislike seeing DB queries like DeployGroup.includes(:environment).order(:environment_id,
> :name).all in the view code.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/302/files#r26600368>.
>